### PR TITLE
feat(blog): Add Intl support for publish date

### PR DESCRIPTION
**Pull request by MarkoOleksiyenko**

### CODE REVIEW

File: demo/src/lib/layout/Published.svelte [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<script>
+	export let date;
+
+	date = Intl.DateTimeFormat('en-US', {dateStyle: 'full'}).format(new Date(date));
```
Comment (quentinderoubaix) [inline demo/src/lib/layout/Published.svelte:4]: maybe it should be:
```typescript
$: dateString = Intl.DateTimeFormat('en-US', {dateStyle: 'full'}).format(new Date(date));
```

**Review by quentinderoubaix — COMMENTED** [review]
thx for the PR @MarkoOleksiyenko, left you a few minor comments

File: demo/src/lib/layout/Published.svelte [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+</script>
+
+<strong class="d-flex mb-2">Published on {date}</strong>
```
[the remark this answers is not in the document]
Comment (MarkoOleksiyenko) [reply demo/src/lib/layout/Published.svelte:7]: without it the mb is not applied

File: demo/src/lib/layout/Published.svelte (hunk 1 above)
Comment (MarkoOleksiyenko) [reply demo/src/lib/layout/Published.svelte:4]: was thinking about the reactivity dangle, but as it is rendered only once not sure that it is actually needed, wdyt?

Comment (quentinderoubaix) [reply demo/src/lib/layout/Published.svelte:4]: It's not really needed indeed... though it provides better code readability.

**Pull request merged**